src/Microsoft.AspNetCore.OData/Results/ResultHelpers.cs

```diff
@@ -59,5 +60,18 @@ public static void AddEntityId(HttpResponse response, Func<Uri> entityId)
                 response.Headers.Add(EntityIdHeaderName, entityId().ToString());
             }
         }
+
+        public static void AddServiceVersion(HttpResponse response, Func<string> version)
```

Comment:
Rename to "AddServiceVersionForNoContentResponse" to indicate that it only handle the case of NoContent response code.

Reply:
Thanks. That's same syntax of the above method “AddEntityId". 
Besides, that's a public method in an internal class. It's not mandatory to add "comments" in an internal class.